groclient/client.py

```diff
@@ -698,6 +698,56 @@ def get_geojson(self, region_id, zoom_level=7):
         """
         return lib.get_geojson(self.access_token, self.api_host, region_id, zoom_level)
 
+    def get_descendant(
+        self,
+        entity_type,
+        entity_id,
+        distance=None,
+        include_details=True,
+    ):
+        """Look up details of all entities level contained by an item/metric/regions.
```

Comment:
I'm not sure I'm understanding the docstring - what does "level" mean? Also, I think it would be nice for users to specify how this differs from get_descendant_regions. Perhaps something like this?

> Look up details of entities contained by an item/metric/region.
>
> Similar to :meth:`~.get_descendant_regions`, but also works on items and metrics. This method has a `distance` parameter (which returns all nested child entities) instead of a `descendant_level` parameter (which only returns child entities at a given depth/level).

We could also modify the docstring for `get_descendant_regions` to just say something like:
> Also see :meth:`~.get_descendant`, which works on items and metrics as well.